.circleci/config.yml

```diff
@@ -266,7 +266,7 @@ jobs:
               export PYTHONPATH=.:$PYTHONPATH
               # This is a flag that enables test_test_baseline_training to work
               export TEST_BASELINE_SMALL=1
-              python setup.py test --addopts -v -s "test/test_baseline_training.py"
+              python setup.py test --addopts "test/test_baseline_training.py"
```

Comment:
python setup.py test --addopts "-s  test/test_baseline_training.py"   #`-s` is for verbose